settings.yml

```
@@ -68,7 +68,7 @@ variables:
   # ------------------------------- #
 
   bicepRegistryDoPublish: true # Set to true, if you would like to publish module templates to a bicep registry
-  bicepRegistryName: adpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
+  bicepRegistryName: skhanadpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
```

Please revert changes to this file
```suggestion
  bicepRegistryName: adpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
```

---

This issue is not resolved. The settings.yml file should not show any changes